pkg/model/proto/model.pb.go

```diff
@@ -3,14 +3,13 @@
 
 package proto
 
+import proto "github.com/golang/protobuf/proto"
```

Comment:
I don't understand why this file was updated since no change was made in model.proto.